### PR TITLE
Make mobilising skills/interests optional

### PR DESCRIPTION
**Pull request by MichaelKohler**
That was easier than I thought :)

### CODE REVIEW

**Comment by MichaelKohler** [comment]
@akatsoulas r?

File: remo/profiles/models.py [hunk 1]
```diff
@@ -216,9 +216,11 @@ class UserProfile(models.Model):
     functional_areas = models.ManyToManyField(
         FunctionalArea, related_name='users_matching')
     mobilising_skills = models.ManyToManyField(
-        MobilisingSkill, related_name='users_matching_skills')
+        MobilisingSkill, related_name='users_matching_skills',
+        blank=True)
```
Comment (akatsoulas) [inline remo/profiles/models.py]: I would suggest to add a default value of `default=''`.

**Comment by akatsoulas** [comment]
Also don't forget to update the migration :)

File: remo/profiles/models.py (hunk 1 above)
Comment (MichaelKohler) [reply remo/profiles/models.py]: done :)

**Comment by akatsoulas** [comment]
After discussing this with @MichaelKohler, I am closing this in favor of #1410

**Pull request closed without merging**